sentiment/bert_sentiment.py

```diff
@@ -0,0 +1,385 @@
+# -*- coding: utf-8 -*-
+"""BERT_sentiment.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1uBQqFUJwsK4rw3t1yBNX3WTu7CJ74VNS
+"""
+
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nna
+import pickle
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import classification_report
+
+device = torch.device("cuda")
+
+"""
+This section grabs the SemEval dataset from Dropbox.
+Requires a dropbox API key.
+"""
+import os.path
+import dropbox
+
+if not os.path.isfile('semeval.zip'):
+  ACCESS_TOKEN = "dhmkFjUUQYAAAAAAAAAAAcRFqu7bkzhtbmjYLVZEbMg4wiyYtytDtXzXkShJT4Kc" 
```

Comment:
Whose access token is this?

Access tokens shouldn't be hardcoded. You can pass it on the command line using argparse.ArgumentParser or load it from a config file that is excluded in the .gitignore.

If this access token is your personal one, I suggest re-generating it if possible since it is here in the PR history.

Reply:
Yea good call, dropbox has already sent me an email saying that they deactivated my API key. I removed the key and update some of the functions to better align with ones in `comparison.py` as they share the same code.

Dropbox was initially used to develop on google colab as a mechanism for persistent storage for saving the model after training since storage in colab runtimes is ephemeral.